src/dashboard-refactor/lists-sidebar/components/lists-search-bar/list-search.ts

```diff
@@ -0,0 +1,49 @@
+import { ListsSearchables, ListsSearchResult } from './types'
+
+export const searchLists = (
+    searchStr: string,
+    searchData: ListsSearchables,
+): ListsSearchResult[] => {
+    const { sources, lists } = searchData
```

Comment:
You might already know, but you can do this object deconstructing in the function params themselves. e.g.: 
```
const searchLists = (
  searchStr: string,
  { sources, lists }: ListSearchables,
): ListSearchResult[] => {
  ...
}
```

Reply:
Good spot, I think I knew I could do it but I wasn't clear on how and so the knowledge didn't surface. Will keep in mind!